docs/software/rpm-development-guide.md

```diff
@@ -247,6 +247,19 @@ where:
 In addition, if the repository contains a file called `rpm/<PROJECT>.spec`, it will be used as the spec file for the build
 (unless overridden in the `osg` directory).
 
+##### Typycal workflow when building out of Git and GitHub repos
+
+1. Fork the repository that you would like to build for
```

Comment:
I think we're missing a step after this one that says something about making and committing changes to the repository. Once you add that, the step that says "repeat steps 2-5" should say "repeat steps 2-5".